### DISCUSSION TITLE
Gnome-Boxes (possible bug) and congratulations to the creator of Spiral Linux for creating this GNU/Linux distribution.

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#14] [General]
Hello, good evening.
First of all forgive my English as it is not my primary language. I speak Spanish. Since a few days ago I am using Spiral Linux with the Mate desktop (version 11.220610). I have not had any problem, only one, and that is that Gnome-Boxes (the version of the repositories) does not work correctly once installed in Spiral Linux (I got an error message saying that the virtualization extensions were not installed when I was going to create a virtual machine), even if the BIOS was well configured to allow virtualization on my computer.
I was searching the Internet, but I found no solution, so I installed Gnome-Boxes through Flatpak and with the Flatpak version it works correctly.
I write this on the one hand to report this possible bug, but mostly to congratulate the developer and creator of Spiral Linux, as I think he is doing a good job. Spiral Linux is one of the distributions I will recommend.
Thanks and best regards.
I hope that a great community of users of this distribution will be created.

**Comment by geckolinux** [comment]
Hi there, thanks for trying SpiralLinux and for your positive comments! Your English is great. I also speak Spanish too if you prefer.

I actually had the same problem with Gnome Boxes from the Debian repositories. It can be fixed by installing:
`qemu-system-x86`
`qemu-utils`
I think those packages should be listed as required for `gnome-boxes`, but SpiralLinux is pure Debian and doesn't modify upstream packages in any way.

**Comment by ghost** [comment]
Thank you for your response and for solving the problem. I figured that maybe there was some qemu related package to install, but I didn't know which one exactly.
Greetings from Spain!

Gracias por responder y por solucionar el problema. Imaginé que tal vez habia que instalar algún paquete relacionado con qemu, pero no sabía cual era exactamente.
Saludos desde España!

**Reply by geckolinux** [reply]
Gracias y saludos!
